argo/common/run_completion_event_unit_test.go

```diff
@@ -0,0 +1,189 @@
+//go:build unit
+
+package common
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Context("RunCompletionEvent.Validate", func() {
+
+	validate, _ = InitialiseValidation()
```

Comment:
should check the error here else if it does happen to fail all you will get in the test output is probably a panic

Reply:
changed in [e78e58f](https://github.com/sky-uk/kfp-operator/pull/352/commits/e78e58f3ef3c6464dd3d9c2fcce26544d39f5512)